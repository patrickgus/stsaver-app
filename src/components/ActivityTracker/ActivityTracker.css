.ActivityTracker {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.ActivityTracker h2 {
  margin: 15px;
}

.ActivityTracker__chart img {
  width: 250px;
}
