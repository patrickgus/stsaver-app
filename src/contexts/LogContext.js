import React, { Component } from "react";

const LogContext = React.createContext({
  LogList: [],
  totalHours: {},
  userId: {},
  error: null,
  setError: () => {},
  clearError: () => {},
  setLogList: () => {},
  setTotalHours: () => {},
  addLog: () => {}
});

export default LogContext;

export class LogProvider extends Component {
  state = {
    logList: [],
    totalHours: {},
    userId: {},
    error: null
  };

  setLogList = logList => {
    this.setState({ logList });
  };

  setTotalHours = totalHours => {
    this.setState({ totalHours });
  };

  setUserId = userId => {
    this.setState({ userId });
  };

  setError = error => {
    console.error(error);
    this.setState({ error });
  };

  clearError = () => {
    this.setState({ error: null });
  };

  addLog = log => {
    this.setLogList([...this.state.logList, log]);
  };

  render() {
    const value = {
      logList: this.state.logList,
      totalHours: this.state.totalHours,
      userId: this.state.userId,
      error: this.state.error,
      setError: this.setError,
      clearError: this.clearError,
      setLogList: this.setLogList,
      setTotalHours: this.setTotalHours,
      setUserId: this.setUserId,
      addLog: this.addLog
    };
    return (
      <LogContext.Provider value={value}>
        {this.props.children}
      </LogContext.Provider>
    );
  }
}
