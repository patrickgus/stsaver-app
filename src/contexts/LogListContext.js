import React, { Component } from "react";

const LogListContext = React.createContext({
  LogList: [],
  error: null,
  setError: () => {},
  clearError: () => {},
  setLogList: () => {}
});

export default LogListContext;

export class LogListProvider extends Component {
  state = {
    logList: [],
    error: null,
  };

  setLogList = logList => {
    this.setState({ logList })
  }

  setError = error => {
    console.error(error)
    this.setState({ error })
  }

  clearError = () => {
    this.setState({ error: null })
  }

  render() {
    const value = {
      logList: this.state.logList,
      error: this.state.error,
      setError: this.setError,
      clearError: this.clearError,
      setLogList: this.setLogList,
    }
    return (
      <LogListContext.Provider value={value}>
        {this.props.children}
      </LogListContext.Provider>
    )
  }
}

