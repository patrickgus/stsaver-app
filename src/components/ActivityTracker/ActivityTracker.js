import React from "react";
import ActivityChart from "../../images/Activity-chart-demo.PNG";
import "./ActivityTracker.css";

function ActivityTracker() {
  return (
    <div className="ActivityTracker">
      <h2>Activity Tracker</h2>
      <div className="ActivityTracker__chart">
        <img src={ActivityChart} alt="placeholder for activity tracker chart" />
      </div>
      <dl className="ActivityTracker__stats">
        <dt>Total Hours:</dt>
        <dd>120</dd>
        <dt>This week:</dt>
        <dd>12</dd>
        <dt>Weekly goal:</dt>
        <dd>50</dd>
      </dl>
    </div>
  );
}

export default ActivityTracker;
