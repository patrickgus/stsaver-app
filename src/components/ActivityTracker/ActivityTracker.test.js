import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import { LogProvider } from "../../contexts/LogContext";
import ActivityTracker from "./ActivityTracker";

describe("ActivityTracker component", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <LogProvider>
        <ActivityTracker />
      </LogProvider>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <LogProvider>
          <ActivityTracker />
        </LogProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
