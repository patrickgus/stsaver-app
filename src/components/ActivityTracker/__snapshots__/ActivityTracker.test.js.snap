// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityTracker component renders the UI as expected 1`] = `
<div
  className="ActivityTracker"
>
  <h3>
    Activity from the past 7 days
  </h3>
  <div
    className="ActivityChart"
  >
    <div
      className="VictoryContainer"
      style={
        Object {
          "height": "100%",
          "pointerEvents": "none",
          "position": "relative",
          "touchAction": "none",
          "userSelect": "none",
          "width": "100%",
        }
      }
    >
      <svg
        height={350}
        role="img"
        style={
          Object {
            "height": "auto",
            "pointerEvents": "all",
            "width": "100%",
          }
        }
        viewBox="0 0 500 350"
        width={500}
      >
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shapeRendering="auto"
            style={
              Object {
                "fill": "transparent",
                "stroke": "#90A4AE",
                "strokeLinecap": "round",
                "strokeLinejoin": "round",
                "strokeWidth": 2,
              }
            }
            vectorEffect="non-scaling-stroke"
            x1={50}
            x2={450}
            y1={300}
            y2={300}
          />
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={128.00121876904328}
              x2={128.00121876904328}
              y1={300}
              y2={50}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={128.00121876904328}
              x2={128.00121876904328}
              y1={300}
              y2={305}
            />
            <text
              direction="inherit"
              dx={0}
              dy={12.825}
              id="chart-axis-0-tickLabels-0"
              x={128.00121876904328}
              y={313}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="middle"
                x={128.00121876904328}
              >
                0.2
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={206.00243753808655}
              x2={206.00243753808655}
              y1={300}
              y2={50}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={206.00243753808655}
              x2={206.00243753808655}
              y1={300}
              y2={305}
            />
            <text
              direction="inherit"
              dx={0}
              dy={12.825}
              id="chart-axis-0-tickLabels-1"
              x={206.00243753808655}
              y={313}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="middle"
                x={206.00243753808655}
              >
                0.4
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={284.0036563071298}
              x2={284.0036563071298}
              y1={300}
              y2={50}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={284.0036563071298}
              x2={284.0036563071298}
              y1={300}
              y2={305}
            />
            <text
              direction="inherit"
              dx={0}
              dy={12.825}
              id="chart-axis-0-tickLabels-2"
              x={284.0036563071298}
              y={313}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="middle"
                x={284.0036563071298}
              >
                0.6
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={362.0048750761731}
              x2={362.0048750761731}
              y1={300}
              y2={50}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={362.0048750761731}
              x2={362.0048750761731}
              y1={300}
              y2={305}
            />
            <text
              direction="inherit"
              dx={0}
              dy={12.825}
              id="chart-axis-0-tickLabels-3"
              x={362.0048750761731}
              y={313}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="middle"
                x={362.0048750761731}
              >
                0.8
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={440.00609384521636}
              x2={440.00609384521636}
              y1={300}
              y2={50}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={440.00609384521636}
              x2={440.00609384521636}
              y1={300}
              y2={305}
            />
            <text
              direction="inherit"
              dx={0}
              dy={12.825}
              id="chart-axis-0-tickLabels-4"
              x={440.00609384521636}
              y={313}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="middle"
                x={440.00609384521636}
              >
                1
              </tspan>
            </text>
          </g>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shapeRendering="auto"
            style={
              Object {
                "fill": "transparent",
                "stroke": "#90A4AE",
                "strokeLinecap": "round",
                "strokeLinejoin": "round",
                "strokeWidth": 2,
              }
            }
            vectorEffect="non-scaling-stroke"
            x1={50}
            x2={50}
            y1={50}
            y2={300}
          />
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={450}
              y1={300}
              y2={300}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={45}
              y1={300}
              y2={300}
            />
            <text
              direction="inherit"
              dx={0}
              dy={5.324999999999999}
              id="chart-axis-1-tickLabels-0"
              x={37}
              y={300}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="end"
                x={37}
              >
                0
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={450}
              y1={251.9969278033794}
              y2={251.9969278033794}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={45}
              y1={251.9969278033794}
              y2={251.9969278033794}
            />
            <text
              direction="inherit"
              dx={0}
              dy={5.324999999999999}
              id="chart-axis-1-tickLabels-1"
              x={37}
              y={251.9969278033794}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="end"
                x={37}
              >
                2
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={450}
              y1={203.9938556067588}
              y2={203.9938556067588}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={45}
              y1={203.9938556067588}
              y2={203.9938556067588}
            />
            <text
              direction="inherit"
              dx={0}
              dy={5.324999999999999}
              id="chart-axis-1-tickLabels-2"
              x={37}
              y={203.9938556067588}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="end"
                x={37}
              >
                4
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={450}
              y1={155.99078341013825}
              y2={155.99078341013825}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={45}
              y1={155.99078341013825}
              y2={155.99078341013825}
            />
            <text
              direction="inherit"
              dx={0}
              dy={5.324999999999999}
              id="chart-axis-1-tickLabels-3"
              x={37}
              y={155.99078341013825}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="end"
                x={37}
              >
                6
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={450}
              y1={107.98771121351768}
              y2={107.98771121351768}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={45}
              y1={107.98771121351768}
              y2={107.98771121351768}
            />
            <text
              direction="inherit"
              dx={0}
              dy={5.324999999999999}
              id="chart-axis-1-tickLabels-4"
              x={37}
              y={107.98771121351768}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="end"
                x={37}
              >
                8
              </tspan>
            </text>
          </g>
          <g
            role="presentation"
          >
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "none",
                  "pointerEvents": "painted",
                  "stroke": "#ECEFF1",
                  "strokeDasharray": "10, 5",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={450}
              y1={59.98463901689709}
              y2={59.98463901689709}
            />
            <line
              role="presentation"
              shapeRendering="auto"
              style={
                Object {
                  "fill": "transparent",
                  "size": 5,
                  "stroke": "#90A4AE",
                  "strokeLinecap": "round",
                  "strokeLinejoin": "round",
                  "strokeWidth": 1,
                }
              }
              vectorEffect="non-scaling-stroke"
              x1={50}
              x2={45}
              y1={59.98463901689709}
              y2={59.98463901689709}
            />
            <text
              direction="inherit"
              dx={0}
              dy={5.324999999999999}
              id="chart-axis-1-tickLabels-5"
              x={37}
              y={59.98463901689709}
            >
              <tspan
                dx={0}
                style={
                  Object {
                    "fill": "#455A64",
                    "fontFamily": "'Roboto', 'Helvetica Neue', Helvetica, sans-serif",
                    "fontSize": 15,
                    "letterSpacing": "normal",
                    "padding": 8,
                    "stroke": "transparent",
                    "strokeWidth": 0,
                  }
                }
                textAnchor="end"
                x={37}
              >
                10
              </tspan>
            </text>
          </g>
        </g>
        <text
          direction="inherit"
          dx={15}
          dy={35.325}
          x={0}
          y={0}
        >
          <tspan
            dx={15}
            style={
              Object {
                "fill": "#252525",
                "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Ser­avek', 'Trebuchet MS', sans-serif",
                "fontSize": 15,
                "parent": Object {
                  "height": "100%",
                  "userSelect": "none",
                  "width": "100%",
                },
                "stroke": "transparent",
              }
            }
            textAnchor="start"
          >
            Hours
          </tspan>
        </text>
        <g
          role="presentation"
        />
      </svg>
      <div
        style={
          Object {
            "height": "auto",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
            "zIndex": 99,
          }
        }
      >
        <svg
          height={350}
          style={
            Object {
              "height": "auto",
              "overflow": "visible",
              "width": "100%",
            }
          }
          viewBox="0 0 500 350"
          width={500}
        />
      </div>
    </div>
  </div>
  <dl
    className="ActivityTracker__stats"
  >
    <dt>
      Total Hours:
    </dt>
    <dd />
  </dl>
</div>
`;
